specification/iotoperations/IoTOperations.Management/models/auth.tsp

```diff
@@ -31,8 +31,7 @@ model DataflowEndpointAuthenticationSasl {
   @doc("Type of SASL authentication. Can be PLAIN, SCRAM-SHA-256, or SCRAM-SHA-512.")
   saslType: DataflowEndpointAuthenticationSaslType;
 
-  @doc("Token secret name.")
-  secretRef: string;
```

Comment:
i don't know if we should have Sasl config inherit from the auth access token model like this. can we keep the `secretRef: string` property explicitly defined here?

Reply:
reverted